secure-api-gateway-ob-uk-rs-resource-store/secure-api-gateway-ob-uk-rs-resource-store-api/src/main/java/com/forgerock/sapi/gateway/rs/resource/store/api/admin/DataCreator.java

```diff
@@ -71,16 +71,8 @@
 import com.forgerock.sapi.gateway.rs.resource.store.repo.mongo.customerinfo.FRCustomerInfoRepository;
 
 import lombok.NoArgsConstructor;
-import uk.org.openbanking.datamodel.v3.account.OBBeneficiary5;
-import uk.org.openbanking.datamodel.v3.account.OBReadBalance1DataBalanceInner;
-import uk.org.openbanking.datamodel.v3.account.OBReadDirectDebit2DataDirectDebitInner;
-import uk.org.openbanking.datamodel.v3.account.OBReadOffer1DataOfferInner;
+import uk.org.openbanking.datamodel.v4.account.*;
```

Comment:
Can you reference the exact classes please.

Reply:
yep, sorry about this, Intellij has this behavior when I import 5 or more classes from the same package.